WordPress/gutenberg
# [RNMobile] Fix background and text colour on pullquote block

Fixes #34358

This PR still needs more testing on my end.

This PR fixes the color selection for the text and background that was removed in https://github.com/WordPress/gutenberg/pull/30951

## Description
Adds back the colours for the text and background. 

## How has this been tested?
1. Load this PR on a mobile simulator (iOS or Android) 
2. Add the pullquote block. 
3. Make a colour selection notice that the text and the background color update as expected.
4. Preview the changes. Notice that they look as expected. 

Testing selfhosted sites.
6. On a WP 5.8 self hosted site. Create a few different variations of the pull quote.
7. Do the pull quotes look as expected in the editor / similar to the preview.  

## Screenshots 

## Types of changes
Bug Fix

## Checklist:
